installation/checkInstallation.m

```diff
@@ -317,23 +318,22 @@
     status = 0; % No need to run on Windows
     return;
 end
-
 binDir = fullfile(findRAVENroot(),'software');
 
-binList = {fullfile(binDir,'blast+','blastp');
-           fullfile(binDir,'blast+','makeblastdb');
-           fullfile(binDir,'cd-hit','cd-hit');
-           fullfile(binDir,'diamond','diamond');
-           fullfile(binDir,'hmmer','hmmbuild');
-           fullfile(binDir,'hmmer','hmmsearch');
-           fullfile(binDir,'GLPKmex','glpkcc');
-           fullfile(binDir,'libSBML','TranslateSBML');
-           fullfile(binDir,'libSBML','OutputSBML');
-           fullfile(binDir,'mafft','mafft-linux64','mafft');
-           fullfile(binDir,'mafft','mafft-mac','mafft');};
+binList = {fullfile(binDir,'blast+','blastp');                  fullfile(binDir,'blast+','blastp.mac');
+           fullfile(binDir,'blast+','makeblastdb');             fullfile(binDir,'blast+','makeblastdb.mac');
+           fullfile(binDir,'cd-hit','cd-hit');                  fullfile(binDir,'cd-hit','cd-hit.mac');
+           fullfile(binDir,'diamond','diamond');                fullfile(binDir,'diamond','diamond.mac');
+           fullfile(binDir,'hmmer','hmmbuild');                 fullfile(binDir,'hmmer','hmmbuild.mac');
+           fullfile(binDir,'hmmer','hmmsearch');                fullfile(binDir,'hmmer','hmmsearch.mac');
+           fullfile(binDir,'GLPKmex','glpkcc.mexa64');          fullfile(binDir,'GLPKmex','glpkcc.mexglx');         fullfile(binDir,'GLPKmex','glpkcc.mexmaci64');
+           fullfile(binDir,'libSBML','TranslateSBML.mexa64');   fullfile(binDir,'libSBML','TranslateSBML.mexglx');  fullfile(binDir,'libSBML','TranslateSBML.mexmaci64');
+           fullfile(binDir,'libSBML','OutputSBML.mexa64');      fullfile(binDir,'libSBML','OutputSBML.mexglx');     fullfile(binDir,'libSBML','OutputSBML.mexmaci64');
+           fullfile(binDir,'mafft','mafft-linux64','mafft.bat');
+           fullfile(binDir,'mafft','mafft-mac','mafft.bat');};
 
 for i=1:numel(binList)
-    [status,cmdout] = system(['chmod +x "' binList{i} '".(mexa64|mexglx|mexmaci64|mac|bat)']);
+    [status,cmdout] = system(['chmod +x "' binList{i} '"']);
```

Comment:
To me it looks like the meaningful change here is the position of the second `"`, which before was before the `.` and now is at the end. I'm also a bit unsure if quotation marks were needed at all - maybe it was a mechanism to deal with any spaces that would be found in the full path of `binDir`.